### ISSUE TITLE
The problem about training results

### ISSUE THREAD

**Issue by xytjcxy** [LinXiLuo/my_delf#3]
Thanks for your training codes! However, I meet a problem when I start training. The problem is that the value of loss function doesn't change around 9. Do you meet the same problems or can you please point out my problems, thanks!

**Comment by xytjcxy** [comment]
@LinXiLuo ，have you solved your problem now?

**Comment by LinXiLuo** [comment]
Hi @xytjcxy
The loss function doesn't change.
Yes, this is the exact problem I have met. 
It is trained well in resnet50 on landmarks, but fail in attention layer training.
I tried lr from 1e-1 to 1e-5, but didn't work.
if any advice, let me know please.

**Comment by xytjcxy** [comment]
To be honest, tf is too difficult and I find the training codes written with pytorch ,[https://github.com/nashory/DeLF-pytorch](url)，maybe you can have a try！
